packages/babel-plugin-transform-flow-comments/test/fixtures/flow-comments/class-prop-values-3/expected.js

```diff
@@ -0,0 +1,7 @@
+class X {
+  constructor() {
+    this.a = 2;
+    this.b = '3';
```

Comment:
See my comment here: https://github.com/babel/babel/pull/4623#issuecomment-250861503

Babel 6.15 completely lost the types too. I'll look into preserving them next.